src/orchestrationclient.ts

```diff
@@ -326,25 +323,27 @@ export class OrchestrationClient {
     }
 
     private getClientData(): OrchestrationClientInputData {
-        const dataKey = this.context.bindings ? Object.keys(this.context.bindings).filter((val: string) => {
-            return Object.keys(new OrchestrationClientInputData(undefined, undefined, undefined)).every((key) => {
-                return this.context.bindings[val].hasOwnProperty(key);
-            });
-        })[0] : undefined;
+        const dataKey = (this.context as IFunctionContext).bindings
```

Comment:
this is a gnarly bit of code. I'd recommend putting in a method and writing some unit tests

Reply:
Done.